src/pages/Privacy.vue

```diff
@@ -0,0 +1,130 @@
+
+                  <h3>Surveys and Evaluation Activities</h3>
+
+                  <p>From time to time ARC may run surveys through Rosie. Surveys may offer you the option to provide your email address for further contact from ARC’s evaluation team.</p>
+
+                  <h3>Mailing Lists and Our Newsletter</h3>
+
+                  <p>You can subscribe to our mailing list by providing your email address at <a href="http://www.alexandrarose.org.uk" class="info" target="_blank">www.alexandrarose.org.uk</a> – ARC will only use email addresses to send relevant information. We will not pass your data on to third parties unconnected with ARC, and users may unsubscribe simply by clicking the link at the foot of our email to them or by requesting to do so by email to <a href="mailto:info@alexandrarose.org.uk" class="info" target="_top">info@alexandrarose.org.uk</a></p>
+
+                  <h3>Disclosure and Transfer of Your Personal Data</h3>
+
+                  <p>We do not sell, rent or otherwise disclose personally identifiable information with any other parties other than as set out in this Privacy Policy. Your data is stored securely in line with data protection law. The Data Protection Act 1998 does however permit certain disclosures without consent when the information is requested for one or more of the following purposes:</p>
+
+                  <ul>
+                      <li>To safeguard national security;</li>
+                      <li>Prevention or detection of crime including the apprehension or prosecution of offenders;</li>
+                      <li>Assessment or collection of tax duty;</li>
+                      <li>Discharge of regulatory functions (includes health, safety and welfare of persons at work);</li>
+                      <li>To prevent serious harm to a third party;</li>
+                      <li>To protect the vital interests of the individual, this refers to life and death situations.</li>
+                  </ul>
+
+                  <p>All requests to provide personal data for one of the above reasons will be specifically authorised by ARC’s Business Administrator.</p>
+
+                  <h3>Use of Cookies and Other Technologies</h3>
+
+                  <p>A Cookie is a small piece of data containing a unique reference which is written to a web-enabled user device such as a computer, smartphone, tablet and/or other device running a web browser capable of connecting you to the internet. ARC uses this technology to collect information about Rosie users in order to provide them with a better experience by recognising each user and informing him/her of any new messages since their previous visit. ARC also uses Google Analytics to gather non-personally identifiable information on our visitors. This information helps us understand how Rosie is being used and is gathered through cookies and code embedded in Rosie.</p>
```

Comment:
We currently only use cookies to provide better user exp. Is keeping login. Not for any data or analytics.